packages/react-form-hooks/src/hooks/validateAll.ts

```diff
@@ -0,0 +1,36 @@
+import {useCallback} from 'react';
+import {FieldDictionary, FieldOutput, FormError, Field} from '../types';
+import {isField} from '../utilities';
+
+export function useValidateAll(fieldBag: {[key: string]: FieldOutput<any>}) {
```

Comment:
I'm worried about performance of these traversal functions in very big forms, but I sort of think we solve that issue when it becomes a problem